generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -344,3 +346,29 @@ class FireSimTraceGenL2Config extends Config(
     outerLatencyCycles = 50) ++
   new WithTraceGenBridge ++
   new FireSimRocketChipConfig)
+
+
+class WithRationalTiles(multiplier: Int, divisor: Int) extends Config((site, here, up) => {
```

Comment:
Is this the right place for this? Or name?
I think there is a similar Config object in rocketchip

Reply:
I'll look for it.

Reply:
I believe we can tear out all the TraceGen related stuff out of firechip. @zhemao do you still want it in here? TraceGenConfigs are now run out of Chipyard.